changelogs/fragments/5588-support-1password-connect.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - onepassword lookup plugin - support 1Password Connect with the opv2 client (https://github.com/ansible-collections/community.general/pull/7116).
```

Comment:
The new options should be mentioned here. You should also mention the onepassword_raw plugin, since that's also affected.